src/safeds/data/tabular/containers/_table.py

```diff
@@ -991,6 +993,27 @@ def transform_column(self, name: str, transformer: Callable[[Row], Any]) -> Tabl
             return self.replace_column(name, result)
         raise UnknownColumnNameError([name])
 
+    def inverse_transform_table(self, transformer: InvertibleTableTransformer) -> Table:
+        """
+        Use a transformer to inverse the transformation of this table.
+
+        Parameters
+        ----------
+        transformer : InvertibleTableTransformer
+            The transformer which is fitted on this table
```

Comment:
The transformer doesn't need to be fitted on this table. This table just needs to have all columns that the transformer changed.